gammapy/cube/make.py

```diff
@@ -116,6 +116,32 @@ def _process_obs(self, obs, selection):
             data = maps_obs[name].quantity.to(self.maps[name].unit).value
             self.maps[name].fill_by_coord(coords, data)
 
+    def make_images(self, spectrum=None, edisp=None):
+        """Create 2D maps by summing over energy axis.
+
+        Parameters
+        ----------
+        spectrum : `~gammapy.spectrum.models.SpectralModel`, default is None
+            Spectral model to use to weight exposure in true energy
+            If None is passed, a power law of photon index -2 is assumed.
+        edisp : `~gammapy.irf.EnergyDispersion`, default is None.
+            Energy Dispersion response to redistribute true energies to reco energies
+            If no energy dispersion is passed, a diagonal one is assumed with identical true and reco
+            energy bins. This is the default behaviour.
+
+        Returns
+        -------
+        images : dict of `~gammapy.maps.Map`
+            the dictionary of images
+        """
+        images = dict()
+        for name, map in self.maps.items():
+            if name == 'exposure':
+                expo_reco = make_map_exposure_reco_energy(map, spectrum, edisp)
+                images[name] = expo_reco.sum_over_axes()
+            images[name] = map.sum_over_axes()
```

Comment:
Move this line into an else block?

I think otherwise, you're overwriting the exposure image with just the sum image like for counts and background?